api/src/main/java/org/openmrs/module/esaudemetadata/MetadataBundleUtils.java

```diff
@@ -6,6 +6,21 @@
 		public static final String WRITE_TEST_ORDER = "Write Test Order";
 		public static final String READ_TEST_ORDER = "Read Test Order";
 		public static final String UPDATE_TEST_ORDER = "Update Test Order";
+		public static final String DELETE_VITALS = "Delete Vitals";
+		public static final String EDIT_VITALS = "Edit Vitals";
+		public static final String READ_ANAMNESIS = "Read Anamnesis";
```

Comment:
in OpenMRS instead of saying **Read** we say **View**

Reply:
@ningosi the privilege names come from https://docs.google.com/spreadsheets/d/1HJioc0wpZUsjjIkVM3t_vpb5UJkTtJ1V1i-VOdA_Px0/edit#gid=0